lm_eval/evaluator.py

```diff
@@ -75,6 +82,11 @@ def simple_evaluate(
 
     assert tasks != [], "No tasks specified"
 
+    if use_updated_scorer:
+        os.environ["USE_UPDATED_SCORER"] = "yes"
```

Comment:
Control params through env vars is dirty hack, but on dev stage it is OK. Further need to investigate the harness tool to do it cleaner